warnsystem/warnsystem.py

```diff
@@ -455,9 +455,9 @@ def _set_log(self, sentry: "Log"):
         """
         Set if the bot should respect roles hierarchy.
 
-        If enabled, a member cannot ban another member above him in the roles hierarchy, like\
+        If enabled, a member cannot ban another member above them in the roles hierarchy, like\
```

Comment:
"a member" is singular here, "him" should stay.

Reply:
English is an annoying language. "Them" can be be used as plural or singular depending on how it's used.